habitat/tasks/rearrange/multi_task/composite_sensors.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
```

Comment:
Are these tasks scripts user facing (will the user be expected to read or modify them)? I think that if this is the case they would need some typing annotations and commands.

Reply:
Agreed, we should be typing everything and trying to document as much as possible.